share/disambiguation.py

```diff
@@ -144,6 +144,32 @@ def _instance_for_node(self, node):
         logger.error('Could not disambiguate %s. Too many results found from %s %s', node.model, all_query, queries)
         raise NotImplementedError('Multiple {0}s found'.format(node.model))
 
+    def _instance_for_subject(self, node):
+        # Subject disambiguation is a bit weird: Match (uri) OR (name AND taxonomy)
+        if 'uri' in node.attrs:
+            try:
+                return node.model.objects.get(uri=node.attrs['uri'])
+            except node.model.DoesNotExist:
+                pass
+
+        name = node.attrs.get('name')
+        if node.related('central_synonym') is None:
+            # Central taxonomy
+            if name:
+                try:
+                    return node.model.objects.get(name=name, central_synonym__isnull=True)
+                except node.model.DoesNotExist:
+                    logger.debug('No %s found with name "%s" in the central taxonomy', node.model, name)
```

Comment:
Not if it's the system user initializing/updating the central taxonomy. Permissions are (somewhat hackily) checked when [accepting changes](https://github.com/CenterForOpenScience/SHARE/pull/674/files#diff-26baa8ae87e868e4bb3653a66e0d26f9R285).